### ISSUE TITLE
Update installation Documentation

### ISSUE THREAD

**Issue by ldrach** [pannal/Sub-Zero.bundle#792]
The install documentation seems a bit outdated. The documentation says to go to 'Channels' to configure Sub-Zero but that does not seem to be the way to do it anymore. Now to configure Sub-Zero you would go to Settings > Plugins.

Old:
![Old Way](https://user-images.githubusercontent.com/40499729/189543418-7c025ff5-8ac4-461c-8233-c99505cb9866.png)

New:
![New Way](https://user-images.githubusercontent.com/40499729/189543575-50662122-76ed-4047-980a-5cf483e19c64.png)

**Comment by MarkRMonaco** [comment]
I agree with this.  The guide was written before the Live TV interface was updated with Plex's own selection of streaming content (back when "Live TV" relied on having a TV tuner either directly connected to the server or via the network; i.e.: HDHomeRun).  While the method you posted above will get you to the configuration section (just like when you use the "gear" icon on the agent selection/configuration screen), it will still not get you to the actual "main" interface of this plugin (as shown in the installation/configuration guide), which I know existed prior to this change (as I used to use it before many of the plugins became depreciated in Plex).
